public/data/job-openings-data.json

```diff
@@ -798,6 +798,69 @@
           "experience": "3 to 8 Years experience"
         }
       ]
+    },
+    {
+      "id": 19,
+      "companyName": "Capermint Technologies Pvt. Ltd.",
+      "location": "Ahmedabad, India",
+      "bio": "Capermint Technologies is a leading Mobile App and Game Development company in India with ISO 9001:2015 Certification & also has a presence in Canada, USA, France & UK. We use the latest and high-quality technologies like AR, VR, AI, XR, Etc to develop apps and games that deliver unique and mesmerizing experiences to users. Capermint Technologies is a global leader in mobile game/application design and development. We create comprehensive games and applications for startups, businesses, and enterprises across various Android and iOS. Our vision is to constantly develop mobile apps and games and grow as a major mobile app and game development company.",
+      "companyLogoUrl": "https://drive.google.com/file/d/1m753gaF5jbl2hXc37ABq2UDy18feq-cw/view?usp=sharing",
```

Comment:
This URL will not work (Company logo will not load), Please refer other jobs object.